.github/workflows/generate.yml

```diff
@@ -0,0 +1,30 @@
+on:
+  pull_request:
```

Comment:
Since the action won't run on PRs anyways, why not just have it run on push to master? 😅 
```suggestion
```

Reply:
done